### ISSUE TITLE
@DATABASE@ is empty when exporting?

### ISSUE THREAD

**Issue by STaRDoGG** [phpmyadmin/phpmyadmin#17858]
### Describe the bug

To save from re-typing, here is a link to the description on StackOverflow: https://stackoverflow.com/q/74254950/553663

### To Reproduce

1. Go to database overview on the server
2. Select 1 more more db's to export
3. Enter @SERVER@ @DATABASE@__%Y-%m-%d_%H-%M-%S as the file template name, so it'll include the actual name of the db in the exported file name.
4. Click the export button

### Expected behavior

The db name is in the exported file name.

### Screenshots

Included in the above SO link.

### Server configuration

Latest PHPMA container in Docker.

### Client configuration

- Browser: Tried in several browsers (Brave, Chrome, Edge)
- Operating system: Win 10, Win 11

### Additional context

Asked on SO 2 days ago and only 16 views so far, with no answer, so now I'm here. =)

**Comment by liviuconcioiu** [comment]
First, if multiple databases are exported, you can't have the database name in the archive filename (which name to choose?). I think is not working because this is how it was intended to work.

Also, there is a bug, when a single database is exported (is missing from the filename) and when multiple databases are exported (`@DATABASE@` isn't used as a replacement for the names - is only added at the end).

https://user-images.githubusercontent.com/25424343/199318762-65514da5-69e4-4cac-bd8b-6c39a70d4102.mp4

@williamdes will tell us if there's a bug or not :).

**Comment by STaRDoGG** [comment]
> First, if multiple databases are exported, you can't have the database name in the archive filename (which name to choose?). I think is not working because this is how it was intended to work.

I agree on that part and is understandable; the only thing is, I usually also check the box saying to export them individually when I select multiple db's, and it still doesn't add the individual db names, which can add a lot of unnecessary confusion because then the file names are just saved as `mysql-etc.sql.gz`, `mysql(1)-etc.sql.gz`, `mysql(2)-etc.sql.gz`, and so on, which means at some later point in time when hunting for a specific db, one needs to decompress each file, and open the sql file in a text editor until they find the db they wanted.

**Comment by YouveGotMeowxy** [comment]
Here's some more discussion on it:

https://gist.github.com/iamphilrae/d538d5b48608de290bfee791df3e6f54?permalink_comment_id=4352266#gistcomment-4352266

**Comment by JLuc** [comment]
Possibly same future enhancement with the `%TABLE%` placeholder when setting "export each table as a separate file"
